commands/general/macro.js

```diff
@@ -0,0 +1,89 @@
+module.exports.load = (client) => {
+  client.commands['macro'] = {
```

Comment:
I don't love the name `macro`. These feel sort of like sub-commands of `help` to me. Any other names come to mind?

Reply:
Maybe `info`?